laboratory-servlet-api-integration/src/it/java/com/wixpress/petri/test/LaboratoryIT.java

```diff
@@ -141,4 +140,13 @@ public void reportsSingleSpecToPetri() throws Exception {
         assertThat(specs.get(0).getScopes(), CoreMatchers.is(Arrays.asList(ValidStubSpecDefinition_1.scopeDefinitions)));
     }
 
+    @Test
+    public void resilientToPetriServerDowntime() throws Exception {
```

Comment:
I don't really agree. I think you're saying that because you know how koboshi works. The test I added meant to say that *laboratory* now has caching and is so resilient to petri server downtime. The test failed before I added anything because laboratory didn't have any sort of caching, now it does have it. I'm not trying to get into the details of koboshi there. WDYT?